### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#183] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente osservazione sull'anello $(\mathbb{Z},+,\cdot)$:
- $a|b \land b|a \Rightarrow a = (\pm\ 1) b$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/caaece2c-a9f4-40c8-80ce-f6a604471929)
